media-gfx/pngcheck/pngcheck-2.3.0-r1.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="verifies the integrity of PNG, JNG and MNG files with internal checksums"
+HOMEPAGE="http://www.libpng.org/pub/png/apps/pngcheck.html"
+SRC_URI="mirror://sourceforge/png-mng/${P}.tar.gz"
+
+LICENSE="HPND GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="sys-libs/zlib"
```

Comment:
not sure if it is need to stay here

Reply:
Hmm, i checked the package and it clearly rdepends on ```sys-libs/zlib```. Or, do you think it should not stay here because it's a system package? I usually only check for package which are part of followin list ```eix --system -c```
```sys-libs/zlib``` isn't included, which is why i haven't removed it.